.github/workflows/bump-patch-version.yml

```
@@ -75,4 +75,13 @@ jobs:
             
             This PR was automatically created by the Bump Patch Version workflow.
+
+            ---
+
+            - [ ] close and re-open:
+
+            Bot PRs don't trigger pr actions... create-pull-request/docs/concepts-guidelines.md at main · peter-evans/create-pull-request
+
+            [create-pull-request/docs/concepts-guidelines.md at main](/peter-evans/create-pull-request/docs/concepts-guidelines.md)
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (1 > 0)

(empty-lines)